postgresql/resource_postgresql_grant_role_test.go

```diff
@@ -58,40 +58,39 @@ func TestRevokeRoleQuery(t *testing.T) {
 	var roleName = "foo"
 	var grantRoleName = "bar"
 
+	expected := fmt.Sprintf("REVOKE %s FROM %s", pq.QuoteIdentifier(grantRoleName), pq.QuoteIdentifier(roleName))
+
 	cases := []struct {
-		resource   *schema.ResourceData
+		resource   map[string]interface{}
 		privileges []string
 		expected   string
 	}{
 		{
-			resource: schema.TestResourceDataRaw(t, resourcePostgreSQLGrantRole().Schema, map[string]interface{}{
+			resource: map[string]interface{}{
 				"role":       roleName,
 				"grant_role": grantRoleName,
-			}),
-			expected: fmt.Sprintf("REVOKE %s FROM %s", pq.QuoteIdentifier(grantRoleName), pq.QuoteIdentifier(roleName)),
+			},
 		},
 		{
-			resource: schema.TestResourceDataRaw(t, resourcePostgreSQLGrantRole().Schema, map[string]interface{}{
+			resource: map[string]interface{}{
 				"role":              roleName,
 				"grant_role":        grantRoleName,
 				"with_admin_option": false,
-			}),
-			expected: fmt.Sprintf("REVOKE %s FROM %s", pq.QuoteIdentifier(grantRoleName), pq.QuoteIdentifier(roleName)),
+			},
 		},
 		{
-			resource: schema.TestResourceDataRaw(t, resourcePostgreSQLGrantRole().Schema, map[string]interface{}{
+			resource: map[string]interface{}{
 				"role":              roleName,
 				"grant_role":        grantRoleName,
 				"with_admin_option": false,
```

Comment:
This case is the same than the one above. You probably want to set it to true even it does not change the generated query.